helpme/helpme.py

```diff
@@ -0,0 +1,1049 @@
+    [{"type": "ipv4", "address": "207.246.75.123", "port": 9735}],
+    '029acd76d581901375281c7489720cea277d25252832dadb6f0327f621673c8096':
+    [{"type": "torv3", "address": "dxu2n6mfoiqzcc7i2mc6bxfcmylcngeoxgxvboidghw4ic5psrfyd6yd.onion", "port": 9735}],
+    '03e5f9d1935c67a029bf0a26af5f63109f4f4c45993a8f45177a8f2f39bcb46144':
+    [{"type": "ipv4", "address": "84.246.200.122", "port": 9735}],
+    '029cc41810f968059892a13c3e08ed8ad2f45267f1545e77c5f850d6b177eb63fa':
+    [{"type": "ipv4", "address": "95.174.125.24", "port": 9735}],
+    '034eedf763483d461bac5fe689f5a3cf599f359a3588a323d8f1f43d92ba12fabc':
+    [{"type": "ipv4", "address": "50.79.227.81", "port": 9735}],
+    '02b5195a2378808d2fc87aa4cb4fb985fb6d459e86f861f383eba4d27a116167b2':
+    [{"type": "ipv4", "address": "54.196.81.194", "port": 9735}],
+    '03b46c5222c373ce95353d1ef1ecc8728cb53f54f4253f129a9062aa35f9a737fd':
+    [{"type": "ipv4", "address": "159.89.230.135", "port": 9735}],
+    '035cc46bd30aff9bad94ff82af52e3d73f980ed29ab7f1274f7efa41759dfc5dac':
+    [{"type": "ipv4", "address": "178.62.81.192", "port": 9735}],
+    '0387e89ea3fa2f73d6ec0293ab59114873db5113bfe0cc83083a4c3dc545e13265':
+    [{"type": "ipv4", "address": "95.31.9.47", "port": 9735}],
+    '0328f46ce2d0fe39ed920d7e846f93ce91b440080b42c08aee4ecc8a0d73974e8c':
+    [{"type": "torv2", "address": "vmc6yv6mu6run6zd.onion", "port": 9735}],
+    '0270dd534325064f5cdb8707672244a51e45f2bea6919e8614a03965e48c106de9':
+    [{"type": "ipv4", "address": "185.200.118.101", "port": 9735}],
+    '0388082ef17afb126db9a0877e5f8a8c1a88aec86107fd1f26c2503c384c570fdc':
+    [{"type": "ipv4", "address": "159.89.29.186", "port": 9735}],
+    '024e7585ab8feaf75628c4438a5290d7a9e3698c8cd51c4a89dc6e81f2d3ae2d42':
+    [{"type": "ipv4", "address": "77.27.181.229", "port": 9745}],
+    '024c7faea76f9437c12f2815afaff712721ffdb612e81c80bada5bcc0f8969f1d8':
+    [{"type": "ipv4", "address": "78.133.43.170", "port": 9735}],
+    '0303ba0fed62039df562a77bfcec887a9aa0767ff6519f6e2baa1309210544cd3d':
+    [{"type": "ipv4", "address": "5.9.150.112", "port": 9735}],
+    '02b252dd1f2769beccd50378ac99319637c6bb988206e1c181bec4bcb10a97f815':
+    [{"type": "ipv4", "address": "118.163.74.161", "port": 19735}],
+    '022cfa8589eb544358e2a94f849d15319e6356a443b4fb11668fdc113692681034':
+    [{"type": "ipv4", "address": "2.238.196.161", "port": 9735}],
+    '030444f1a4807f97612c8c6d5f9e4c9e75b96059dd99ccb14b6b39bb962104cf4d':
+    [{"type": "ipv4", "address": "0.0.0.0", "port": 9735}],
+    '0207a77f9012103a9e557f14e80f445946002b2752390b86554d7ca13eb1e6e0e9':
+    [{"type": "ipv4", "address": "5.102.145.58", "port": 9735}],
+    '032720f82414c8ce006d67004398b99697ae3f076c090745d6440ae3c3ade2af45':
+    [{"type": "ipv4", "address": "66.70.135.170", "port": 19735}],
+    '02b0ee0e06112db6929b1bc861d3659c608c61e5ab5899e02d417d1986ae0ab485':
+    [{"type": "ipv4", "address": "204.48.26.93", "port": 9735}]
+}
```

Comment:
Would probably be best to have this as a separate JSON file, instead of inline. That way it could get regenerated on a semi-regular basis.